examples/todo-list/src/models/todo-list-image.model.ts

```diff
@@ -29,3 +29,7 @@ export class TodoListImage extends Entity {
     super(data);
   }
 }
+
+export interface TodoListImageRelations {}
```

Comment:
`TodoListImage` already have some relations defined (I think it belongs to TodoList?), can you add navigational properties for these relations to this interface please?